openmetadata-airflow-apis/src/openmetadata/api/apis_metadata.py

```diff
@@ -72,6 +72,18 @@
             },
         ],
     },
+    {
+        "name": "delete_dag",
+        "description": "Delete a DAG in the Web Server from Airflow database and filesystem",
+        "http_method": "POST",
```

Comment:
should we make this DELETE method instead?

Reply:
that's a good point, let me check and change it

Reply:
updated